grimlock-examples/src/main/scala/commbank/grimlock/test/cmp.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+#
+# Copyright 2018 Commonwealth Bank of Australia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+for file in $(cd demo.scalding; ls)
```

Comment:
minor. Simpler if we do this?
```suggestion
for file in grimlock-core/*;
```